hack/install_golangci.sh

```diff
@@ -1,5 +1,17 @@
-#!/bin/bash -e
+#!/bin/bash
+
+if [ -z "$VERSION" ]; then
+	echo \$VERSION is empty
+	exit 1
+fi
+
+if [ -z "$GOBIN" ]; then
+	echo \$GOBIN is empty
+	exit 1
+fi
+
 $GOBIN/golangci-lint --version | grep $VERSION
 if [ $?  -ne 0 ]; then
+	set -e
```

Comment:
kill this line.

Reply:
See comment above. This is command that should not fail, all above are fine.

Reply:
...something like this:
```bash
linter=$GOBIN/golangci-ling
if [ ! -x $linter ]; then
    echo "linter unavailable: $linter"
    exit 1
fi
if ! grep "$VERSION" <<($linter --version); then
    echo "linter is the wrong version"
    $linter --version
    exit 1
fi